index.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,900;1,100;1,300&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Harmattan&family=Roboto:ital,wght@0,900;1,100;1,300&display=swap"
+      rel="stylesheet"
+    />
+    <title>Gideon's Portfolio</title>
+    <script
+      src="https://kit.fontawesome.com/9947fbd823.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+
+  <body>
+    <header class="template5-main-mobile">
+      <div class="app-bar">
+        <div class="time"><img src="Time.png" alt="time" /></div>
+        <a href="#" class="name">Gidiz Logo</a>
+      </div>
+      <i class="fa-solid fa-bars"></i>
+      <div class="ellipse1"></div>
+      <div class="ellipse2"></div>
+
+      <section class="headline">
```

Comment:
- [ ] Exactly here starts the headline section, which should be in the `section` tag - you did it.

Reply:
Noted

Reply:
Done